### ISSUE TITLE
Storage API

### ISSUE THREAD

**Issue by ramonck** [ForstaLabs/libsignal-node#1]
How could we use the storage api to store the information where we want?

Do you have any examples?

**Comment by mayfield** [comment]
Good question;  The code is fork/port from https://github.com/WhisperSystems/libsignal-protocol-javascript , so the usage is fairly simillar (some minor name changes mostly as I recall).  We should take this as a call to action to update the docs a bit about this (to have parity with the web based lib).  In the meantime you can reference one of our other libs which uses this library for doing the higher level protocols (and storage) here... https://github.com/ForstaLabs/librelay-node/blob/master/src/storage/index.js  

Basically that's a module that just exports the various methods expected by `libsignal`.  It's passed as an argument to functions like `libsignal.SessionCipher(storage)`. For example, https://github.com/ForstaLabs/librelay-node/blob/master/src/message_receiver.js#L211
